pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -258,12 +260,20 @@ func (fs *filesystem) newInode(nodeID uint64, attr linux.FUSEAttr) *kernfs.Dentr
 	i := &inode{fs: fs, NodeID: nodeID}
 	creds := auth.Credentials{EffectiveKGID: auth.KGID(attr.UID), EffectiveKUID: auth.KUID(attr.UID)}
 	i.InodeAttrs.Init(&creds, linux.UNNAMED_MAJOR, fs.devMinor, fs.NextIno(), linux.FileMode(attr.Mode))
+	// TODO(gvisor.dev/issue/1193): This should be handled in Init,
+	// unfortunately we need it for this implementation now.
+	atomic.StoreUint64(&i.size, attr.Size)
 	i.OrderedChildren.Init(kernfs.OrderedChildrenOptions{})
 	i.dentry.Init(i)
 
 	return &i.dentry
 }
 
+// Ino overrides the Ino() from InodeAttrs.
+func (i *inode) Ino() uint64 {
```

Comment:
Why do we need this?

Reply:
To avoid confusion, as it has been causing bugs twice.

Will remove